sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md

```diff
@@ -4,11 +4,7 @@
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added DisableDetectScaleUnits to AssetConversionOptions
```

Comment:
IIRC, we should be adding a section for ## 0.3.0-beta.6 (Unreleased)

Reply:
I think it will generate by the bot, see this PR https://github.com/Azure/azure-sdk-for-net/pull/31081 , so to avoid duplicates, I will leave it to the bot.